### PR TITLE
Updated community-guidelines.md

### PR DESCRIPTION
**Pull request by iAyon**
Made some slight changes to make the community guidelines more clear & descriptive.

### CODE REVIEW

**Comment by alanna** [comment]
I am assigning this to @piamancini because she originally wrote this section.

File: about/community-guidelines.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Community Guidelines
 
-We are a community based on trust, transparency, and contribution. In order to keep it that way, we expect everyone to act in accordance with these guidelines, including all Collectives, Backers, Sponsors, and our own Open Collective team.
+We are a community based on trust, transparency, contribution and we try to ensure that all participants in the community have the best possible experience. We are all here to help each other learn, grow our skillsets and have a good time!
+
+These Community guidelines take into consideration the behavior as members of the community in any form of correspondence .
```
Comment (piamancini) [inline about/community-guidelines.md]: Please remove the first line - it's redundant and unclear.

File: about/community-guidelines.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+This isn't an exhaustive list of things that you can't do. Rather, it will be a matter of healthy discussion if you take these points in the spirit in which it's intended.
```
Comment (piamancini) [inline about/community-guidelines.md]: Please remove this too. I agree that we can add a point about discussion, but we enforce the community guidelines. Not observing them is not ok. Happy to  review an alternative wording about the discussion.

File: about/community-guidelines.md [hunk 3]
```diff
@@ -12,23 +17,35 @@ Collectives are groups of people with a shared mission. People may come and go,
 
 ### **Be inclusive**
 
-The reason we can do great things together is that each of us bring a different perspective. Whoever you are, you have a unique perspective and unique skills to contribute.
+The reason we can do great things together is that each of us bring a different perspective. Whoever you are, you have a unique perspective and unique skills to contribute. Moreover as we strive towards becoming a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to, members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion and mental and physical ability.So be welcoming and inclusive .
```
Comment (piamancini) [inline about/community-guidelines.md]: Please remove 'Moreover as', it is grammatically incorrect in this sentence.To make it correct, sentence needs another clause: for example: "as we strive (...) we decided to (...)".  You can start with "We strive(...)

Needs a space after the full stop and before 'So be'

File: about/community-guidelines.md [hunk 4]
```diff
@@ -12,23 +17,35 @@ Collectives are groups of people with a shared mission. People may come and go,
...
 
 ### **Be respectful** 
 
-We have zero tolerance for trolling or harassment. Being a jerk will not get you a place in this community; it will turn people away. We don’t want that noise in the system.
+We have zero tolerance for trolling or harassment. Being a jerk will not get you a place in this community; it will turn people away. We don’t want that noise in the system.We know that not all of us will agree on certain points all the time, but we strongly believe that disagreement is a strict no excuse for poor behavior and poor manners amongst the community. We as a community might experience some or other frustration now and then due to disagreements, but we cannot afford to make that frustration turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one.
```
Comment (piamancini) [inline about/community-guidelines.md]: Needs a space after full stop and before 'We know'

Please remove "on certain points" it's not adding much and it's making the sentence a mouthful 

Please change to "(...) but we strongly believe that disagreement is not an excuse for poor behavior. It's is important to remember that a community (...)" Remove the frustration bit in the middle, it's too lengthly and the spirit of your sentence is captured in the paragraph anyway.

File: about/community-guidelines.md [hunk 5]
```diff
@@ -12,23 +17,35 @@ Collectives are groups of people with a shared mission. People may come and go,
...
+If something’s wrong, let us know. We believe in self-managing systems, so we don’t police or moderate much. The only this works is by sharing the effort of noticing what's going on. Contact us any time:  [support@opencollective.com](mailto:support@opencollective.com).  Moreover we understand that everyone doesn't know everything, and nobody is expected to be perfect in our community. Asking questions and putting your views avoids many problems down the lane, as we get to know where we are lagging and how can we make this community a better place to learn & explore .
+
+### Be cautious with your words and actions. 
+
+Being an ever growing community , we always try to conduct ourselves professionally. Therefore you are required to not insult or put down other participants. Harassment and other exclusionary behavior is strictly not acceptable and should be reported as soon as possible. This includes but is not limited to:
```
Comment (piamancini) [inline about/community-guidelines.md]: Please remove "Therefore(...)" we already say that above. 

We can leave the "Harassment (...) " it's a good addition.

**Review by piamancini — CHANGES_REQUESTED** [review]
@ayonroy2000 Thank you so much for your suggestions! They are greatly appreciated. 
I've left some comments and requested some changes.

File: about/community-guidelines.md (hunk 1 above)
Comment (iAyon) [reply about/community-guidelines.md]: Okay maam .

File: about/community-guidelines.md (hunk 2 above)
Comment (iAyon) [reply about/community-guidelines.md]: Sure .

File: about/community-guidelines.md (hunk 3 above)
Comment (iAyon) [reply about/community-guidelines.md]: Thanks for pointing these out. Will do the needful.

File: about/community-guidelines.md (hunk 4 above)
Comment (iAyon) [reply about/community-guidelines.md]: Sure .

File: about/community-guidelines.md (hunk 5 above)
Comment (iAyon) [reply about/community-guidelines.md]: Cool

**Comment by iAyon** [comment]
@piamancini have done the needful . Please review it maam .

**Comment by iAyon** [comment]
@piamancini any updates ma`am ?

**Comment by contraexemplo** [comment]
I'm closing this pull request as it's been open for almost a year and our documentation has changed a lot in the meantime.

**Pull request closed without merging**